### DISCUSSION TITLE
Is there a way to auto convert snake_case column names to their equivalent camelCase with `@map`

### DISCUSSION THREAD

**Discussion by TheCelebrimbor** [prisma/prisma#7428] [Q&A]
Suppose I have a database with a LOOOOOOT of tables and columns already in `snake_case`, I use `introspect` to pull the db to the prisma schema. 

Is there a way that prisma can auto `camelize` the column names and add a `@map` attribute without me having to manually do it to each and every column ?

**Comment by ryands17** [comment]
@TheCelebrimbor 👋 

Currently Prisma doesn't do this out of the box, but we have a request for this here and a workaround provided in there as well that you could check.

https://github.com/prisma/prisma/issues/1934

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
